apps/settings/js/panels/improve_browser_os/panel.js

```diff
@@ -1,37 +1,103 @@
 define(function(require) {
   'use strict';
 
-  var SettingsCache = require('modules/settings_cache');
   var SettingsPanel = require('modules/settings_panel');
+  var SettingsCache = require('modules/settings_cache');
+
+  const METRICLEVELSETTING = 'metrics.selectedMetrics.level';
+  const DOGFOODSETTING = 'debug.performance_data.dogfooding';
+  const SHAREDATASETTING = 'debug.performance_data.shared';
 
   return function ctor_improveBrowserOS() {
     return SettingsPanel({
       onInit: function(panel) {
         var appUsage = panel.querySelector('#appUsageByRegion');
-        var sharePrefDataToggle =
-          panel.querySelector('#menuItem-sharePerformanceData gaia-checkbox');
+        var metricsBasic = panel.querySelector('#metrics-basic');
+        var metricsEnhanced = panel.querySelector('#metrics-enhanced');
+        var metricsNone = panel.querySelector('#metrics-none');
+        this.levelsMap = {
+          'Basic': 'Basic',
+          'Enhanced': 'Enhanced',
+          'None': 'None'
+        };
+        appUsage.addEventListener('click', this);
+        metricsBasic.addEventListener('click', this);
+        metricsEnhanced.addEventListener('click', this);
+        metricsNone.addEventListener('click', this);
 
-        appUsage.addEventListener('click', this._openAppUsage.bind(this));
+        var metricsLevel;
+        SettingsCache.getSettings((results) => {
```

Comment:
you just need single  SettingsCache.getSettings((results)  and you can parse metricsLevel, dogFooding, dogfood from the results[]